pkg/lifecycle/lifecycle.go

```diff
@@ -496,8 +496,9 @@ type Rule struct {
 
 // Configuration is a collection of Rule objects.
 type Configuration struct {
-	XMLName xml.Name `xml:"LifecycleConfiguration,omitempty" json:"-"`
-	Rules   []Rule   `xml:"Rule"`
+	XMLName                 xml.Name `xml:"LifecycleConfiguration,omitempty" json:"-"`
+	Rules                   []Rule   `xml:"Rule"`
+	ExpiryIgnoreReplication string   `xml:"-,omitempty" json:"-"`
```

Comment:
is this name appropriate - how does `mc` change look like it?

Reply:
the setting is `mc ilm default <alias>/<bucket> --expire-ignore-replication on|off`